tools/templates/IotDeviceBindingTemplate/dotnet_new_iot-device-binding_csharp/tests/_IotDeviceBinding.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I would remove the Tests from the template given that they are not required today.

Reply:
I thought about that, but IMHO it seems like it would be easier for the user to delete compared to creating if needed.  I can remove if you think that is best for now.

Reply:
if that's the case, then perhaps we should just add a big comment on the .csproj and the test file saying that these are not required and may be deleted if desired.

Reply:
Agreed.  I'm not quite sure where you are asking to notify,

So in the _IotDeviceBindingTest.cs file... add something like:

```csharp
[Fact]
public void Test1()
{
   // NOTE: THIS TEST PROJECT IS NOT REQUIRED AND CAN BE DELETED IF NOT USED.
}
```